scipy/special/_support_alternative_backends.py

```diff
@@ -90,10 +90,12 @@ def _chdtr(xp, spx):
         return None
 
     def __chdtr(v, x):
-        res = xp.where(x >= 0, gammainc(v/2, x/2), 0)
-        i_nan = ((x == 0) & (v == 0)) | xp.isnan(x) | xp.isnan(v)
-        res = xp.where(i_nan, xp.nan, res)
-        return res
+        res = gammainc(v / 2, x / 2)  # this is almost all we need
```

Comment:
For performance, could return here for other backends. Hopefully doesn't matter - array API support is currently behind an environment variable, and hopefully  google/jax#20507 will likely be resolved before this is fully supported.